code/modules/arcade/prize_datums.dm

```diff
@@ -111,6 +111,12 @@ GLOBAL_DATUM_INIT(global_prizes, /datum/prizes, new())
 	typepath = /obj/item/ammo_box/caps
 	cost = 30
 
+/datum/prize_item/firecracker
+	name = "Firecracker Grenade"
```

Comment:
\improper to match naming standardization.

Reply:
Not needed in datums as this is directly grabbed to put in the list of the prize machine!